xcfs/Sources/build/main.swift

```diff
@@ -0,0 +1,65 @@
+// use it from root folder:
+// `swift run --package-path xcfs build [all, awk, tar, ios_system, ...]`
+
+import Foundation
+import FMake 
+
+OutputLevel.default = .error
+
+// TODO: We can add more platforms here
+let platforms: [Platform] = [.iPhoneOS, .iPhoneSimulator]
+
+let allSchemes = [
+    "ios_system",
+
+    "awk",
+    "tar",
+    "curl_ios", 
+    "files",
+    "shell",
+    "ssh_cmd",
+    "tar",
+    "text",
+    ]
+
+let args = ProcessInfo.processInfo.arguments 
+
+var schemes: [String] 
+if args.count > 1 && args[1] != "all" {
+    schemes = args[1].components(separatedBy: ",")
+} else {
+    schemes = allSchemes 
+}
+
+var checksums: [(file: String, value: String)] = []
+
+for scheme in schemes {
+    try xcxcf(
+        dirPath: ".build",
+        project: "ios_system",
+        scheme: scheme,
+        platforms: platforms.map { ($0, excludedArchs: []) }
+    )
+
+    try cd(".build") {
+        let zip = "\(scheme).xcframework.zip"
+        try sh("zip -r \(zip) \(scheme).xcframework")
+        let chksum = try sha(path: zip)
```

Comment:
I think you should use [`swift package compute-checksum`](https://developer.apple.com/documentation/swift_packages/distributing_binary_frameworks_as_swift_packages) instead of `shasum` because [its checksum implementation may be changed in the future](https://github.com/apple/swift-evolution/blob/master/proposals/0272-swiftpm-binary-dependencies.md#checksum-computation).

Reply:
Good point

Reply:
But I will monitor that change. I don't know why this `swift package compute-checksum` requires manifest to do it's job.

Reply:
Hi, 
the line `let chksum = try sha(path: zip)` does not work with swift 5 (the error message says: `Illegal instruction`).

Reply:
@holzschu 
Local or GitHub actions? How I can repro that?

Reply:
I have only tested local actions. I assume it affects all package building using Fmake. To reproduce: clone this repository, type `swift run --package-path xcfs build`.

Reply:
Local is fine, see logs:
https://gist.github.com/yury/5cfb388d69bfaf271c3c8ee39824cfbc

```
yury@mbpm1 ~ % swift --version
swift-driver version: 1.26.9 Apple Swift version 5.5 (swiftlang-1300.0.31.1 clang-1300.0.29.1)
Target: x86_64-apple-macosx12.0
```

Reply:
Thanks for testing. I'm puzzled: what could be causing the `Illegal instruction` error? I'll keep searching.
```
zip --symlinks -r ios_system.xcframework.zip ios_system.xcframework
Swift/ErrorType.swift:200: Fatal error: Error raised at top level: FMake.BuildError.unexpectedStatusCode
./createXcFrameworks.sh: line 2: 51765 Illegal instruction: 4  swift run --package-path xcfs build
```